rls-analysis/src/lib.rs

```diff
@@ -239,7 +239,7 @@ impl<L: AnalysisLoader> AnalysisHost<L> {
     }
 
     pub fn get_def(&self, id: Id) -> AResult<Def> {
-        self.with_analysis(|a| a.with_defs(id, |def| def.clone()))
+        self.with_analysis(|a| a.with_defs(id, std::clone::Clone::clone))
```

Comment:
That's in the [prelude](https://doc.rust-lang.org/std/prelude/v1/index.html) so we don't need to write a fully qualified path here:

```suggestion
        self.with_analysis(|a| a.with_defs(id, Clone::clone))
```

Reply:
fixed in latest commit